README.md

```diff
@@ -25,6 +25,10 @@ You must configure the `REACT_APP_MAPBOX_ACCESS_TOKEN` environment variable in a
 nx serve frontend
 ```
 
+#### Style
```

Comment:
```suggestion
## Design
```

Reply:
Would me more appropriate as a level 2 title later in the document.
We could also mentions the company behind the design.

Reply:
> Resolved via dffa02fd1592b5c6aab448313bf01d2ce68c2486